tests/gradients/test_finite_difference.py

```diff
@@ -0,0 +1,608 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the gradients.finite_difference module.
+"""
+import pytest
+
+from pennylane import numpy as np
+
+import pennylane as qml
+from pennylane.gradients import finite_diff, finite_diff_stencil, generate_shifted_tapes
+
+
+class TestStencil:
+    """Tests for the get_stencil function"""

```

Comment:
Ha, so you did have another name before for this function? I find that one more intuitive!

Reply:
I've now renamed the function to `finite_diff_coeffs` :)